js/src/views/Web/web.js

```diff
@@ -0,0 +1,209 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+
+import styles from './web.css';
+
+export default class Web extends Component {
+  static contextTypes = {
+    api: PropTypes.object.isRequired
+  }
+
+  state = {
+    token: null,
+    isLoading: true,
+    displayedUrl: this.lastAddress(),
+    url: this.lastAddress()
+  };
+
+  handleUpdateUrl = (url) => {
+    try {
+      window.localStorage.setItem('parity-web-lastAddress', url);
```

Comment:
Could use `store` and not ned the catch block. In addition, would make the key a constant of the form

```
const LS_LAST_ADDRESS = '_parity::webLastAddress';
```

While we are still wildly inconsistent in naming of localStorage keys, @ngotchac started the `_parity::` movement so it would make sense for new keys to conform.